emails/views.py

```diff
@@ -435,6 +441,13 @@ def _sns_message(message_json):
     return response
 
 
+def _record_receipt_verdicts(message_json, stage):
+    for key, value in message_json['receipt'].items():
+        if key.endswith('Verdict'):
+            value = value['status']
+            incr_if_enabled(f'{stage}_{key}_{value}')
```

Comment:
```suggestion
            histogram_if_enabled(key, value, stage)
```
I think this is a good application for Markus's [histogram](https://markus.readthedocs.io/en/latest/usage.html#markus.main.MetricsInterface.histogram) ([sample](https://markus.readthedocs.io/en/latest/metricsoverview.html?highlight=histogram#getting-statistical-distributions-timing-histogram) scenarios to use histogram). 
Doing this will help answer question on “Does the different verdicts and their result correlate to checks passing/failing?” but we will need to pass multiple `stage`s so we don't duplicate the data point. Since that may be hard to do I think doing:
```
            histogram_if_enabled(stage, key, value)
```
Which means we can ask the same question like:
"In a stage of a process (`stage`), what are the distribution of different verdict types (`key`) broken down by types of verdict results (`value`s)." and can be easier to see if the combinations of verdicts being available or not will also impact the likelihood the email pass/fail-ing the different stage. I reached out to William (willkg) to see if this would be a good way to format the data.

Reply:
Yeah, I did get to talk to him and we think it's best to stick with `incr` but with tags on the data.
```python
metrics.incr(
    "relay.emails.status.checkbounce",
    value=1,
    tags=["dkimverdict:pass", "dmarcverdict:fail", ...]
)

metrics.incr(
    "relay.emails.verdicts.dkim",
    value=1,
    tags=["status:checkbounce"]
)

# Then in grafan, you can graph that with something like:
from relay.emails.status.checkbounce
select field(value) sum()
group by time($__interval) tag:dkimverdict fill(null)
```
Something like this will give us data on:
- each stage/process and how many verdicts were present and pass/failed.
- for verdict what was the process that passed (not necessary, but it's another way to see the previous bullet point but easier to see patterns for a given verdict instead of stage)